libqtile/core/manager.py

```diff
@@ -59,61 +57,67 @@
 from libqtile.widget.base import _Widget
 
 
-def handle_exception(loop, context):
-    if "exception" in context:
-        logger.error(context["exception"], exc_info=True)
-    else:
-        logger.error("exception in event loop: %s", context)
-
-
 class Qtile(CommandObject):
     """This object is the `root` of the command graph"""
     def __init__(
         self,
         kore,
         config,
-        eventloop,
         no_spawn=False,
         state=None
     ):
-        self._restart = False
+        self.core = kore
         self.no_spawn = no_spawn
+        self._state = state
 
-        self.mouse_position = (0, 0)
+        self._stopped_event = None
+        self.should_restart = False
 
-        self.core = kore
-        self.config = config
-        libqtile.init(self)
+        self._drag = None
+        self.mouse_map = None
+        self.mouse_position = (0, 0)
 
         self.windows_map = {}
         self.widgets_map = {}
         self.groups_map = {}
         self.groups = []
+        self.dgroups = None
+
         self.keys_map = {}
         self.current_chord = False
-
         self.numlock_mask, self.valid_mask = self.core.masks
 
+        self.current_screen = None
+        self.screens = []
+
+        libqtile.init(self)
+
+        self._event_loop = None
+        self.server = IPCCommandServer(self)
+        self.config = config
+        # config can potentially be replaced by the default config
+        self.config = self.load_config(config)
```

Comment:
I'd suggest any of the two:

```suggestion
        self.config = self.load_config(config)
```

```suggestion
        self.config = config
        self.load_config()
```

Reply:
Or this:

```suggestion
        self.load_config(config)
```